src/tests/hello.test.js

```diff
@@ -0,0 +1,6 @@
+
+describe('A hello world test that we will delete', () => {
```

Comment:
Hey we could delete it, _or_ we could always leave this in as a test of the testing system itself!

Reply:
Ok that's a fair point -- renamed things so that would be the case.